### ISSUE TITLE
ToC item with slash not detected properly

### ISSUE THREAD

**Issue by arthurlutz** [sindresorhus/awesome-lint#99]
I am getting the following error : 

```
 ToC item "🤳📱 Phones / Tablets" link "#-phones--tablets" not found                                          remark-lint:awesome-toc
```
on https://github.com/arthurlutz/awesome-software-running-on-old-hardware#readme 

https://github.com/arthurlutz/awesome-software-running-on-old-hardware#contents points to https://github.com/arthurlutz/awesome-software-running-on-old-hardware#-phones--tablets which works in my browser.
